__tests__/app.test.js

```diff
@@ -56,3 +58,16 @@ describe("server errors", () => {
       });
   });
 });
+
```

Comment:
You need a Status 200, array of article objects (including comment_count, excluding body).